app/src/main/java/org/simple/clinic/editpatient/EditPatientEffectHandler.kt

```diff
@@ -250,12 +246,14 @@ class EditPatientEffectHandler @AssistedInject constructor(
     return savePatientEffects
         .filter(::isAlternativeIdAdded)
         .flatMapCompletable { savePatientEffect ->
-          userAndCurrentFacility()
-              .flatMap { (user, facility) ->
+          userSession
+              .loggedInUser()
```

Comment:
We have `loggedInUserImmediate()`, so we can probably use that over here?

Reply:
We could, I didn't make this change here since it was not related to the PR.

Reply:
I'll make this change since we can edit this while we are here.

Reply:
Done.